test/built-ins/Function/prototype/bind/15.3.4.5-15-5.js

```diff
@@ -5,16 +5,16 @@
 es5id: 15.3.4.5-15-5
 description: >
     Function.prototype.bind - The [[Configurable]] attribute of length
-    property in F set as false
+    property in F set as true
 ---*/
 
         var canConfigurable = false;
         var hasProperty = false;
         function foo() { }
         var obj = foo.bind({});
         hasProperty = obj.hasOwnProperty("length");
-        delete obj.caller;
+        delete obj.length;
```

Comment:
omg, why were we deleting obj.caller here? 

Thanks for finding this false positive.

This is also a reminder to convert these legacy tests to use the property helpers.

Btw, a reference for the length descriptor from [the specs](https://tc39.github.io/ecma262/#sec-function.prototype.bind)

```
8. Perform ! DefinePropertyOrThrow(F, "length", PropertyDescriptor {[[Value]]: L, [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true}).
```